src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Refactoring/ExtractToCodeBehindTest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Refactoring
+{
+    class ExtractToCodeBehindTest
```

Comment:
I assume you'll add something here before this leaves draft but thought I'd point it out in case it slipped past you.